simpleviewinc/tap-events-force
# ZEN-692: Docker Updates

**Ticket**: [ZEN-692](https://simpleviewtools.atlassian.net/jira/software/c/projects/ZEN/boards/53?modal=detail&selectedIssue=ZEN-692)

## Goal

* Provide the UK team with a docker environment to run the app on windows, without requiring them to setup the `keg-cli`... which doesn't work on windows anyway.

## Updates
* `Dockerfile`
  * Implemented a basic dockerfile that does the job
* `README.md`
  * Update the readme with the docker commands
* Removed the keg-cli container files that aren't needed anymore
* Removed some test files that were never used for keg-herkin

## Testing
* Pull down this branch
* Run `yarn doc:build`
* Then `yarn doc:start`
* Verify the app starts and is accessible at http://localhost:19006/
* Also make sure that the app hot-reloads when you make a change to a file
